### PR TITLE
Migrate some components to Go modules

### PR DESCRIPTION
**Pull request by franpog859**
**Description**

Changes proposed in this pull request:

- migrate Framefrog's components to Go modules

I couldn't migrate `application-connector-tests` to Go Modules as it imports `event-source` which doesn't support it yet. The same thing with `compass-runtime-agent` test which imports `application-broker` which doesn't support Go Modules yet

**As a follow-up:**

- [ ] migrate `application-registry-tests` in the follow-up PR
- [ ] delete fork from the dependencies in the follow-up PR

### CODE REVIEW

File: tests/application-operator-tests/Dockerfile [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-
 WORKDIR $SRC_DIR
+
 COPY . $SRC_DIR
 
 COPY go.mod .
 COPY go.sum .
-
 RUN go mod download
+RUN go mod vendor
```
[the remark this answers is not in the document]
Comment (franpog859) [reply tests/application-operator-tests/Dockerfile]: Sure. I deleted it

**Review by majakurcius — APPROVED** [review]
🚀

**Comment by franpog859** [comment]
/test pre-master-kyma-gke-compass-integration
[previous log](https://status.build.kyma-project.io/view/gcs/kyma-prow-logs/pr-logs/pull/kyma-project_kyma/9214/pre-master-kyma-gke-compass-integration/1297898370169311233)

**Review by Szymongib — APPROVED** [review]

**Pull request merged**